test/specs/scenarios/document.js

```diff
@@ -222,34 +222,40 @@ export const documentScenarios = async (lang) => {
         documentSelector.clickOnDrivingLicenceIcon()
         countrySelector.selectSupportedCountry()
         countrySelector.clickSubmitDocumentButton()
+        // first upload attempt should return an error if image quality is detected
+        // FIXME: Image quality errors are only returned for cut-off images in these tests!
+        // We should be able to define the error type for each request
```

Comment:
Just to be sure I understand this right, was the API only returning cut-off image error even for the glare test image hence why it's only using the cut-off test image now? Or that the tests' flow need to be fixed?

Reply:
A same question from me here

Reply:
Currently, the mock server is returning responses based on the file name used in the tests. So very time we use `identity_card_with_cut-off.png` and `identity_card_with_cut-off_glare.png` the mock server returns an image quality error. This is not ideal because in real life the Onfido API returns an error or warning depending on the error type specified in the request. The SDK requests an image quality `error` for the first two attempts and a `warn` for any subsequent image quality validation. I'm not going to change the behaviour of the mock server right now because we are still discussing possible changes to our testing architecture.

Reply:
Actually we can just modify the mock server a bit to adapt to this new requirement. But I agree that it's still under discussion so the current solution is good enough

Reply:
Yeah, agree that we can leave it as it is now & hold off making further changes to the mock server pending our discussion.

Reply:
Cool, will create a ticket to remind us about this